examples/cr3-event-example.c

```diff
@@ -0,0 +1,138 @@
+/* The LibVMI Library is an introspection library that simplifies access to
+ * memory in a target virtual machine or in a file containing a dump of
+ * a system's physical memory.  LibVMI is based on the XenAccess Library.
+ *
+ * Copyright 2011 Sandia Corporation. Under the terms of Contract
```

Comment:
No need for this copyright notice for Sandia as you don't work for them

Reply:
fixed.